src/VisualStudio/Core/Def/Implementation/Experimentation/KeybindingResetDetector.cs

```diff
@@ -113,9 +121,9 @@ private void InitializeCore()
                 return;
             }
 
-            _resharperExtensionEnabled = extensionEnabled != 0;
+            _resharperExtensionInstalledAndEnabled = extensionEnabled != 0;
 
-            if (_resharperExtensionEnabled)
+            if (_resharperExtensionInstalledAndEnabled)
```

Comment:
I would just use extensionInstalled to see whether you need to hook up the command target.